### PR TITLE
Add footer

### PR DESCRIPTION
**Pull request by dimensi0n**
Related to #2

### CODE REVIEW

File: src/components/layout/Footer.vue [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            <a href="https://twitter.com/darling_web" target="_BLANK" class="soli"><i class="so fab fa-twitter"></i></a>
+            <a href="https://www.facebook.com/mydarlingweb.dev.5" target="_BLANK" class="soli"><i class="so fab fa-facebook"></i></a>
+            <a href="https://github.com/My-Darling-Web" target="_BLANK" class="soli"><i class="so fab fa-github"></i></a>
+            <a href="https://discord.gg/YTA53Cy" target="_BLANK" class="soli"><i class="so  fab fa-discord"></i></a>
+            <p>Tous droits réservés. | myDarlingWeb ©</p>
+        </div>
+    </footer>
+</template>
+
+<style lang="postcss">
```
Comment (Webd01) [inline src/components/layout/Footer.vue]: We are using [tailwindcss](https://tailwindcss.com), do you think that you can use it here? Thanks.

File: src/App.vue [hunk 2]
```diff
@@ -3,6 +3,10 @@
     <transition name="fade">
       <router-view/>
     </transition>
+    <div>
+      <br/>
```
Comment (Webd01) [inline src/App.vue]: <br /> is not needed here.

**Comment by dimensi0n** [comment]
Ok

**Comment by Webd01** [comment]
Thanks to you!

**Pull request merged**